app/assets/stylesheets/components/_organizations.scss

```diff
@@ -0,0 +1,63 @@
+.organization-details {
+  h2, h3, h4, h5 {
+    margin: 0;
+    font: inherit;
+  }
+
+  h2, h4, h5 {
+    font-weight: 700;
+  }
+
+  h2{
+    font-size: 20pt;
+  }
+
+  body {
+    line-height: 1;
+  }
+
+  .border-above{
+    border-top:1px solid #bbb;
+    padding-top:8pt;
```

Comment:
Properties should be ordered border-top, margin-bottom, padding-top<br>Colon after property should be followed by one space